src/feedback/FeedbackForm.css

```diff
@@ -42,17 +68,44 @@
     Open Sans,
     sans-serif
   );
-  resize: vertical;
+
+  resize: none;
```

Comment:
This could be frustrating if someone wants to write a lot.

Reply:
Unfortunately necessary for now, since the UX of resizing a textarea that's a fixed height from the bottom right felt super weird. We could invest in an auto-resizing textarea in future maybe?